backend/model/stock.py

```diff
@@ -0,0 +1,22 @@
+import os
+from matplotlib.style import use
+import psycopg2
+
+class StockDAO:
+
+     def __init__(self):
+        self.conn = psycopg2.connect(os.environ["DATABASE_URL"], sslmode="require")
+
+   
+#####################################################################################################
+#                                       CRUD OPERATIONS                                             #
+#####################################################################################################
+
+def getUserSavedStocks(self, userid):
+    cursor = self.conn.cursor()
+    query = "select symbol from  monitors as M ,  \
+        stock as S where M.userid = %s and s.stockid = M.stockid;"
+    cursor.execute(query, (userid,))
+    stocks = cursor.fetchone()
```

Comment:
I don't remember how the fetchone() method works exactly, but just making an assumption based on its name and what I think I remember, doesn't it returns only the first record of the result of the query or something like that?

Reply:
fixed